src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -245,7 +246,12 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(
   // TODO (Alex): Setting require_available == force_spillback is a hack in order to
   // remain bug compatible with the legacy scheduling algorithms.
   int64_t best_node_id = scheduling_policy_->HybridPolicy(
-      resource_request, force_spillback, force_spillback,
+      resource_request,
+      scheduling_strategy.scheduling_strategy_case() ==
+              rpc::SchedulingStrategy::SchedulingStrategyCase::kSpreadSchedulingStrategy
+          ? 0.0
+          : RayConfig::instance().scheduler_spread_threshold(),
+      force_spillback, force_spillback,
```

Comment:
Do we do round robin always in default as well?

Reply:
I limit the scope of this PR to only add “SPREAD”, no round robin fix (next PR).